Task2.rb

```diff
@@ -0,0 +1,13 @@
+require 'benchmark'
+fib = 1
+second = 0
+sum = 0
+while fib <= 4000000 do
+  fib = fib + second
+  second = fib - second
+  if fib % 2 == 0 then
```

Comment:
You can use `.zero?` instead of `== 0`. It's not necessary, just for the knowledge.
https://www.reddit.com/r/ruby/comments/50qns8/zero_instead_of_0/